zephyr/src/supp_main.c

```diff
@@ -214,6 +227,12 @@ static int z_wpas_remove_interface(const char* ifname)
 		return -1;
 	}
 
+	if (z_wpas_get_iface_count() == 0) {
+		send_wifi_state_event(ifname, NET_EVENT_WPA_SUPP_CMD_NOT_READY);
+	}
+
+	send_wifi_state_event(ifname, NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVED);
```

Comment:
So, shouldn't we now need a status along with this? Success/Failure?

Reply:
At this point we can be assured that interface removal has been handled successfully. In case of failure, we exit the routine at a prior point.

Reply:
No, but shouldn't we inform the applications that are waiting that interface removal has failing? They are stuck at `removing`?

Reply:
Done

Reply:
Done.